scss/4-elements/_forms.scss

```diff
@@ -123,8 +123,8 @@ textarea,
 
   .cads-radio-button-checkmark {
     @include cads-interactive-element();
-    @include cads-transition-animation(none);
 
+    transition: none; /* stylelint-disable property-disallowed-list */
```

Comment:
Calling `cads-transition-animation(none)` was a long way of saying `transition: none`